library/src/main/java/com/bumptech/glide/load/resource/bitmap/GlideBitmapFactory.java

```diff
@@ -72,7 +76,10 @@ public static Bitmap decodeFileDescriptor(
 
   /**
    * Returns a decoded bitmap for the input stream, ensuring that any associated gainmap is decoded
-   * without errors on Android U if it is a valid gainamp.
+   * without errors on Android U if it is a valid gainmap.
```

Comment:
Could you document what happens if there are errors with the gainmap - what does this method return then?